docs/apis/avalanchego/apis/p-chain.md

```diff
@@ -1522,12 +1552,17 @@ curl -X POST --data '{
 
 ### platform.getTotalStake
 
-Get the total amount of nAVAX staked on the Primary Network.
+Get the total amount of nAVAX staked on the requested subnet.
 
 #### **Signature**
 
 ```sh
-platform.getTotalStake(subnetID: string) -> {stake: int}
+platform.getTotalStake({
+    subnetID: string
+}) -> {
+    stake: int
+    weight: int
+}
```

Comment:
We should add explanation for the returned data field.

Reply:
Both `stake` and `weight` will always be equal.